geonode_mapstore_client/client/js/actions/gnresource.js

```diff
@@ -44,6 +48,45 @@ export function setResource(data) {
     };
 }
 
+/**
+* edit the title resource in the state
+* @memberof actions.gnresource
+* @param {string} title resource
+*/
+export function editTitleResource(title) {
+
+    return {
+        type: EDIT_TITLE_RESOURCE,
+        title
+    };
+}
+
+/**
+* edit the abstract resource in the state
+* @memberof actions.gnresource
+* @param {string} abstract resource
+*/
+export function editAbstractResource(abstract) {
+
+    return {
+        type: EDIT_ABSTRACT_RESOURCE,
+        abstract
+    };
+}
+
+/**
+* edit the image resource in the state
+* @memberof actions.gnresource
+* @param {string} image resource
+*/
+export function editImageResource(image) {
+
+    return {
+        type: EDIT_IMAGE_RESOURCE,
```

Comment:
We are using title and abstract in other actions and the names seem aligned with the properties of reaources, so could we use thumbnail or thumbnailUrl instead?

Reply:
do you talk about to rename the action in something like editThumbnailResource, right?

Reply:
yes, I do